stock_inventory/models/stock_inventory.py

```diff
@@ -0,0 +1,193 @@
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class InventoryAdjustmentsGroup(models.Model):
+    _name = "stock.inventory"
+    _description = "Inventory Adjustment Group"
+    _order = "date desc, id desc"
+
+    name = fields.Char(
+        required=True, default="Inventory ", string="Inventory Reference"
+    )
+
+    date = fields.Datetime(default=lambda self: fields.Datetime.now())
+
+    state = fields.Selection(
+        [("draft", "Draft"), ("in_progress", "In Progress"), ("done", "Done")],
+        default="draft",
+    )
+
+    location_ids = fields.Many2many(
+        "stock.location", string="Location", domain="[('usage', '=', 'internal')]"
+    )
+
+    product_selection = fields.Selection(
+        [
+            ("all", "All Products"),
+            ("manual", "Manual Selection"),
+            ("category", "Product Category"),
+            ("one", "One Product"),
+            ("lot", "Lot Serial Number"),
```

Comment:
String = "Lot" would be cleaner here?

Reply:
I would say `Lot/Serial Number` to use Odoo naming

Reply:
@DavidJForgeFlow Maybe an 'owner' management too ?

Reply:
I agree with Lot/Serial Number

Reply:
@rousseldenis what do you mean with 'owner'? Add a responsible field?
The other changes are done, thanks!

Reply:
This one : https://github.com/odoo/odoo/blob/15.0/addons/stock/models/stock_quant.py#L76